classes/i18n/translation/LocaleBundle.php

```diff
@@ -114,6 +116,17 @@ public function setEntries(array $paths): void
         $this->translator = null;
     }
 
+    /**
+     * Returns last cache key, which is useful for LocaleUITranslations cache key to ensure invalidating cache when po files changes
+     *
+     */
+    public function getLastCacheKey(): string
+    {
+        return $this->lastCacheKey ?? $this->_getCacheKey();
```

Comment:
I would move the `_getCacheKey()` implementation to here and remove the private method.

Reply:
I think its better to keep them separate as for getTranslator its important to not get lastCacheKey, but always fresh one. 
While getLastCacheKey has different purpose to provide last cache key to outside world.